### PR TITLE
[hotfix] [docs] fix first example in table API documentation

### PR DESCRIPTION
**Pull request by knaufk**

### CODE REVIEW

File: docs/content/docs/dev/table/common.md [hunk 1]
```diff
@@ -50,26 +50,26 @@ tableEnv.createTemporaryTable("SourceTable", TableDescriptor.forConnector("datag
...
 // Emit a Table API result Table to a TableSink, same for SQL result
-TableResult tableResult = table2.insertInto("SinkTable").execute();
+TableResult tableResult = table1.insertInto("SinkTable").execute();
 ```
 {{< /tab >}}
 {{< tab "Scala" >}}
 ```scala
 import org.apache.flink.table.api._
-import org.apache.flink.connector.datagen.table.DataGenOptions
+import org.apache.flink.connector.datagen.table.DataGenConnectorOptions;
```
Comment (matriv) [inline docs/content/docs/dev/table/common.md:72]:
```suggestion
import org.apache.flink.connector.datagen.table.DataGenConnectorOptions
```

File: docs/content.zh/docs/dev/table/common.md [hunk 2]
```diff
@@ -52,26 +52,27 @@ tableEnv.createTemporaryTable("SourceTable", TableDescriptor.forConnector("datag
...
 // Emit a Table API result Table to a TableSink, same for SQL result
-TableResult tableResult = table2.executeInsert("SinkTable");
+TableResult tableResult = table2.insertInto("SinkTable").execute();
 ```
 {{< /tab >}}
 {{< tab "Scala" >}}
 ```scala
 import org.apache.flink.table.api._
-import org.apache.flink.connector.datagen.table.DataGenOptions
+import org.apache.flink.connector.datagen.table.DataGenConnectorOptions;
```
Comment (matriv) [inline docs/content.zh/docs/dev/table/common.md:74]:
```suggestion
import org.apache.flink.connector.datagen.table.DataGenConnectorOptions
```

File: docs/content.zh/docs/dev/table/common.md [hunk 3]
```diff
@@ -52,26 +52,27 @@ tableEnv.createTemporaryTable("SourceTable", TableDescriptor.forConnector("datag
...
+import org.apache.flink.table.api.bridge.scala.StreamTableEnvironment
```
Comment (matriv) [inline docs/content.zh/docs/dev/table/common.md:75]: Why do you need that?

**Review by matriv — COMMENTED** [review]
@knaufk thx for updating the examples. I left some comments.
Also, I don't see `table1` being used in either java or scala examples, unless I'm missing something.

File: docs/content.zh/docs/dev/table/common.md (hunk 3 above)
Comment (knaufk) [reply docs/content.zh/docs/dev/table/common.md]: I don't. Removed.

File: docs/content/docs/dev/table/common.md (hunk 1 above)
Comment (knaufk) [reply docs/content/docs/dev/table/common.md]: Thanks. Fixed.

**Comment by knaufk** [comment]
>Also, I don't see table1 being used in either java or scala examples, unless I'm missing something.

Yes, I think, either table1 or table2 are unused in these examples.

**Comment by matriv** [comment]
@knaufk Then shouldn't we remove the SQL example and keep only the TableAPI?
@twalthr what do you think?

**Comment by MartijnVisser** [comment]
This PR is subsumed by https://github.com/apache/flink/pull/19681

**Pull request closed without merging**